docs/en/ufs/HDFS.md

```diff
@@ -246,3 +246,17 @@ alluxio.master.mount.table.root.option.alluxio.underfs.version=1.2
 Alluxio v{{site.ALLUXIO_RELEASED_VERSION}} supports the following versions of HDFS as a valid argument of mount option `alluxio.underfs.version`:
 
 - Apache Hadoop: 1.0, 1.2, 2.2, 2.3, 2.4, 2.5, 2.6, 2.7, 2.8, 2.9, 3.0, 3.1
+
+### Use Hadoop Native Library
+
+Hadoop comes with native library that provides better performance and additional features to its Java implementation.
```

Comment:
Suggestions:

- "hadoop provides a native library" or "comes with **a** native library"
- "features **compared** to its java implementation"

Reply:
Updated.